pages/creating-microservices.md

```diff
@@ -11,17 +11,35 @@ sitemap:
 
 Microservices are a type of JHipster application, that have no front-end (the Angular front-end must be generated on a [gateway]({{ site.url }}/api-gateway/)), and which work with the [JHipster Registry]({{ site.url }}/jhipster-registry/) to be configured, discovered, and managed.
 
-<h2 id="generating_entities">Generating entities in a microservices architecture</h2>
+<h2 id="entities">Entities in a microservices architecture</h2>
+
+The user entity is not accessible in the microservice when using JWT or DTO, only when OAuth2 is use, there is a mechanism in the microservice to extract user data from the token and save it to the microservice's database. So for JWT and DTO you can not use and/or define a relationship with the User entity because it exists only in the gateway database.
```

Comment:
```suggestion
The User entity is not generated in a microservice when using JWT or DTO. When OAuth 2.0 is used, there is a mechanism in the microservice to extract user data from the token and save it to the microservice's database. So for JWT and DTO you can not use and/or define a relationship with the User entity because it exists only in the gateway's database.
```